src/Controls/src/Core/HandlerImpl/RadioButton/RadioButton.Tizen.cs

```diff
@@ -76,6 +75,55 @@ static View BuildTizenDefaultTemplate()
 				VerticalOptions = LayoutOptions.Center
 			};
 
+			object outerEllipseVisualStateLight;
+			object outerEllipseVisualStateDark;
+			object checkMarkVisualStateLight;
+			object checkMarkVisualStateDark;
+
+			// First, Light/Dark App themes for outer ellipse, then for the check mark.
+			// Then check for older themecolor.
+			// If nothing set, use the defaults for light/dark mode.
+			if (Application.Current.TryGetResource(RadioButtonOuterEllipseStrokeLight, out var outerLight) &&
```

Comment:
I realize it's a bit funky because of needing to test that old key.  Is there a way we can consolidate this and the similar code below this block into an extension method?
 
 ```C#
 if (normalEllipse.TrySetAppTheme(
	lightResourceKey: RadioButtonOuterEllipseStrokeLight,
	darkResourceKey: RadioButtonOuterEllipseStrokeDark,
	bindableProperty: Ellipse.StrokeProperty,
	defaultDark: SolidColorBrush.Black,
	defaultLight: SolidColorBrush.White,
	out var outerLight,
	out var outerDark))
```

I also wonder if we want to make this if statement an `OR` vs an `AND`. 

If the user has only specified `RadioButtonOuterEllipseStrokeLight` then we probably still want this logic to run and then we'd just set the other side of the `AppThemeBinding` to whatever the default is?